src/php/Prose/FromHost.php

```diff
@@ -341,7 +341,7 @@ public function getScreenSessionDetails($sessionName)
         $log = usingLog()->startAction("get details about screen session '{$sessionName}' on host '{$this->args[0]}' from Storyplayer");
 
         // are there any details?
-        $cmd = "screen -ls | grep -E '[[:digit:]]+.{$sessionName}[[:space:]]' | awk -F. '{print \\\$1}'";
+        $cmd = "screen -ls | grep -E '[[:digit:]]+.{$sessionName}[[:space:]]' | awk -F. '{print $1}'";
```

Comment:
when using double quotes the php parser tries to resolve the variables that starts with the $ symbol.
A safer approach should be:
$cmd = 'screen -ls | grep -E '[[:digit:]]+.{$sessionName}[[:space:]]' | awk -F. \'{print $1}\'';